README.md

```diff
@@ -33,170 +33,140 @@ The latest stable releases are (were) versions that are recommended for producti
 | 6.3      |  [6.3.25](https://github.com/apple/foundationdb/releases/tag/6.3.25) |   Unsupported |
 
 - `Supported` branches are those we actively maintain and will publish new patch releases.
-- `Bug fixes` are branches we still accept bug fixes into the branch, but may not publish newer patch releases. The community can build the latest release binaries if needed and is encouraged to upgrade to the `Supported` branches.
+- `Bug fixes` are branches where we still accept bug fixes, but may not publish newer patch releases. The community can build the latest release binaries if needed and is encouraged to upgrade to the `Supported` branches.
```

Comment:
Should we add 7.4 above? Probably as Pre-release (or experimental).

Reply:
Not quite sure to include it here, since the motivation for this patch is mostly related to building FDB locally. @jzhou77  your thougts?

Reply:
Well, since this is more focused on the build process, let's wait for the 7.4 officially release.